requirements-flake8.txt

```
@@ -2,6 +2,7 @@ flake8==3.8.2
 flake8-bugbear==20.1.4
 flake8-comprehensions==3.3.0
 flake8-executable==2.0.4
+-e https+git://github.com/malfet/flake8-coding.git#egg=flake8-coding
```

What's the reason for using this instead of the PyPI library?
```suggestion
flake8-coding==1.3.2
```

---

Because flake8-coding does not allow skipping annotation for files that do not have utf-8 characters

---

Gotcha, makes sense; I think at the time I posted the comment, you hadn't yet pushed https://github.com/malfet/flake8-coding/commit/1106d953dde998ab6dd7f951efc804dea562d609